### ISSUE TITLE
with -m audio it doesn't grab both sub files

### ISSUE THREAD

**Issue by BlastBolt5** [crunchy-labs/crunchy-cli#141]
**Describe the bug**
when merging audio the subtitle for the secondary lang isn't downloaded
**To Reproduce**
Steps / command to reproduce the behavior:
```
$ ./crunchy-cli archive --skip-existing -m audio -s en-US -o 'Anime/{audio}/{series_name}/Season {season_number}/{series_name} - S{season_number}E{relative_episode_number} - {title}.mkv' https://www.crunchyroll.com/series/GKEH2G428/bofuri-i-dont-want-to-get-hurt-so-ill-max-out-my-defense
```

**Expected behavior**
When merging video the subtitle file is downloaded so there is both a dubbed subtitle file and subbed subtitle file, but this is not the case when merging audio

**Screenshots**
-m auto
![Screenshot from 2023-02-16 04-02-25](https://user-images.githubusercontent.com/64578489/219333735-9d7c8d31-d1b0-4758-af83-53167e180730.png)
-m audio
![Screenshot from 2023-02-16 04-03-04](https://user-images.githubusercontent.com/64578489/219333769-ffd8b61b-68f9-4dcb-b9d1-4cb51dae0dcb.png)

**Client (please complete the following information):**
 - OS: arch linux
 - Version: build from commit 96b259c

**Additional context**
i haven't tried this on dev-8, using latest cause the --skip-existing

**Comment by ByteDream** [comment]
This is the case because with `-m auto` two video tracks are created and the subtitles are for each video. With `-m audio` only one video track is present and therefore only one subtitle track for this one video is required.

**Comment by BlastBolt5** [comment]
the issue is even tho it downloads the Japanese audio, it downloaded the sub for the english version which in bofuri for example the english sub titles are only for translating on screen text not spoken language, is there any way to work around this?

**Comment by ByteDream** [comment]
Fixed/added in [v3.0.0-dev.12](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.0.0-dev.12).
